app/ide-desktop/lib/content/src/index.ts

```diff
@@ -180,60 +180,60 @@ class Main implements AppRunner {
     }
 
     main(inputConfig?: StringConfig) {
-        contentConfig.OPTIONS.loadAll([app.urlParams()])
-        const isUsingAuthentication = contentConfig.OPTIONS.options.authentication.value
-        const isUsingNewDashboard =
-            contentConfig.OPTIONS.groups.featurePreview.options.newDashboard.value
-        const isOpeningMainEntryPoint =
-            contentConfig.OPTIONS.groups.startup.options.entry.value ===
-            contentConfig.OPTIONS.groups.startup.options.entry.default
-        if ((isUsingAuthentication || isUsingNewDashboard) && isOpeningMainEntryPoint) {
-            const hideAuth = () => {
-                const auth = document.getElementById('dashboard')
-                const ide = document.getElementById('root')
-                if (auth) {
-                    auth.style.display = 'none'
-                }
-                if (ide) {
-                    ide.hidden = false
+        const unrecognized = contentConfig.OPTIONS.loadAll([app.urlParams()])
+        if (unrecognized.length > 0) {
+            app.showConfigOptions(contentConfig.OPTIONS, unrecognized)
+        } else {
+            const isUsingAuthentication = contentConfig.OPTIONS.options.authentication.value
+            const isUsingNewDashboard =
+                contentConfig.OPTIONS.groups.featurePreview.options.newDashboard.value
+            const isOpeningMainEntryPoint =
+                contentConfig.OPTIONS.groups.startup.options.entry.value ===
+                contentConfig.OPTIONS.groups.startup.options.entry.default
+            if ((isUsingAuthentication || isUsingNewDashboard) && isOpeningMainEntryPoint) {
+                const hideAuth = () => {
+                    const auth = document.getElementById('dashboard')
+                    const ide = document.getElementById('root')
+                    if (auth) {
+                        auth.style.display = 'none'
```

Comment:
could you remove the HTML completely instead of hiding it? It helps with perf debugging of the app.